index.bs

```diff
@@ -193,7 +195,27 @@ Presentation API Requirements {#requirements-presentation-api}
 Remote Playback API Requirements {#requirements-remote-playback}
 ----------------------------------------------------------------
 
-Issue(3): Requirements for Remote Playback API.
+1.  A [=controlling user agent=] must be able to find out whether there is at
+    least one compatible [=remote playback device=] available for a given
+    {{HTMLMediaElement}}, both instantaneously and continuously.
+
+2.  A controlling user agent must be able to to [=initiate remote playback=] of
+    an {{HTMLMediaElement}} to a compatible remote playback device.
+
+3.  The controlling user agent must be able send media data and text tracks
```

Comment:
Send media data?  Like for remoting?  Hmm... I have not included any messages like that so far.  Should I make a PR to do so?   I guess we could just use the streaming messages that landed recently.

Reply:
I wasn't intending to suggest we were tackling remoting just yet (i.e. real time streaming of data from a media element).  I'll update the language to specifically address the flinging case before merging.